README.md

```diff
@@ -1,50 +1,41 @@
-2. Familiarization with HTML5 Semantic Elements.
+1. Explain the historical reasoning behind semantic elements
+2. Identify specific HTML5 semantic elements
  
-## A Better Way To Markup Content
+## Explain the Historical Reasoning Behind Semantic Elements
 
-<iframe width="640" height="480" src="//www.youtube.com/embed/V28xLFEYaSQ?rel=0&modestbranding=1" frameborder="0" allowfullscreen></iframe>
-
-*Note: Slides for this lecture video are provided in the resources at the bottom of this lesson.*
-
-### A Brief History
-
-At the dawn of HTML, developers had but one common element that was used to create generic boxes and that my friend was the `<div>` element. Pages layouts at that time then became a sea of divs. This worked, but we needed a way to label them to distinguish one div from another. This was largely done with id and class names such as 
+When developers first began defining containers to structure HTML, they had only one generic element available to them: the `div`. Creating complex page layouts then required dozens of `div` elements that were often difficult to organize or locate within the code. They needed a way to distinguish one `div` from another, which led to the use of `id` and `class` names to specify what they intended the `div` to contain.
 
 ```html
 <div id="header">
   <div class="wrapper">...</div>
 </div>
 ```
 
-When the W3C (the organization that oversees the specifications for HTML & CSS) started writing the specification for HTML5 they wanted to create new elements that would eliminate the need to label so many divs and make our code more readable for developers and more descriptive to browsers. Along with the help of Google, hundreds of thousands of websites were scanned and all the most common id and class names were recorded. It turned out that many developers were in fact using the same names to label their elements. Such as `id="header"` or `"footer"`, `"main"`, `"nav"`, `"section"`, `"article"`, `"aside"`, etc. For each of these most common labels we now have elements we can use instead. So instead of
+When the W3C (the organization that oversees the specifications for HTML and CSS) started writing the specification for HTML5 they wanted to create new elements that would eliminate the need to label so many `div` elements. The goals were to make the code more readable for developers and more descriptive for browsers. It turned out that many developers were already using the same names to label their elements, such as `id="header"` or `"footer"`, `"main"`, `"nav"`, `"section"`, `"article"`, `"aside"`, etc. So HTML5 provided semantic elements that explicitly described those functions for developer to use instead.
+
+How does this look in practice? If, for example, we wanted to define a section of our HTML document to contain header content, we once used to have to identify a `div` as our header section.
 
 ```html
 <div id="header">...</div>
 ```
 
-we can use
+Now we can use the `header` element.
```

Comment:
now, instead,

Reply:
"Instead" closes a paragraph just above and I thought it would be too repetitive to include it here again.